easybuild/easyconfigs/d/deepdiff/deepdiff-6.7.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PythonBundle'
+
+name = 'deepdiff'
+version = '6.7.1'
+
+homepage = 'https://zepworks.com/deepdiff/current/'
+description = """DeepDiff: Deep Difference of dictionaries, iterables and almost any other object recursively."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('hatchling', '1.18.0'),  # for setuptools-scm
```

Comment:
No longer needed thanks to #19777

Reply:
I see another deepdiff here #19648 (to be fair, I think I copied files from there to adapt them to CUDA and forgot deepdiff and tensorboardX were not from me)
But I don't see how #19777 would solve the requirement for deepdiff. If I remove deepdiff and tensorboardX from this PR and leave Pytorch-Lightning as is, would that be ok ?

Reply:
@NPotvin the comment of @boegel refers to the line above it: the `hatchling` dep is no longer needed for `deepdiff`.
but it should actually be removed from his own PR https://github.com/easybuilders/easybuild-easyconfigs/pull/19648 (unless you copy it into this PR).